src/routers/RouteRouter.js

```diff
@@ -26,7 +26,8 @@ class RouteRouter extends ApplicationRouter<Array<Object>> {
             uid,
         } = req.query;
 
-        const routes = await RouteUtils.getRoutes(destinationName, end, start, departureTimeQuery, arriveBy);
+        const arriveByBool = arriveBy === '1';
+        const routes = await RouteUtils.getRoutes(destinationName, end, start, departureTimeQuery, arriveByBool);
```

Comment:
I think we should make a version of `RouteRouter` so that this is a POST request as well, same comment regarding better boolean encoding. We can create it on the backend first, and client can push it out in a following update –– can we make a Git issue for this?